Clarification of Trust Anchor Examples (#271)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

Transparency Services MUST maintain Registration Policies.

Transparency Services MUST also maintain a list of trust anchors, which SHOULD be used by Relying Parties to authenticate Issuers, and which MAY be included in a Registration Policy statement.

> For instance, a trust anchor could be an X.509 root certificate, pointing to an OpenID Connect identity provider, or any other COSE-compatible trust anchor.

Reviewer: ```suggestion
For instance, a trust anchor could be an X.509 root certificate, a pointer to an OpenID Connect identity provider, or any other COSE-compatible trust anchor.
```

> For instance, a trust anchor could be an X.509 root certificate, the URL pointing to an OpenID Connect identity provider combined with the use of the Web PKI, or a JSON Web Key Set containing the public keys of Issuers.

Reviewer 2: Not sure what this improves.  The previous was more generic and does not get us into JSON debates.

Author: The reason for the change: Nobody knows what this means "any other COSE compatible PKI trust anchor".

I tried to clarify this. 

The statement about "discovery URL of an OpenID Connect identity provider" is also meaningless.

> For instance, a trust anchor could be an X.509 root certificate, the URL pointing to an OpenID Connect identity provider combined with the use of the Web PKI, or a JSON Web Key Set containing the public keys of Issuers.

Reviewer 3: ```suggestion
For instance, a trust anchor could be an X.509 root certificate, pointing to an OpenID Connect identity provider, or any other COSE-compatible trust anchor.
```

Reviewer 4: I would be happy to approve if this change is integrated.

Reviewer 2: So would I

Author: Give me examples of "any other COSE-compatible trust anchor". 

When the sentence starts with "for instance" or "for example" it has to actually give specific examples rather than leaving the reader puzzled about what you actual mean.

I have worked on COSE for years and I don't know what you are referring. How would any other reader know what you mean?
